turtle.js

```diff
@@ -68,108 +71,102 @@ let turtle = _defaultTurtle();
  * draw the turtle and the current image if `redraw` is `true`.
  * for complicated drawings it is much faster to turn `redraw` off.
 */
-function drawIf() {
-    if (turtle.redraw) draw();
-}
+function drawIf() { if (turtle.redraw) draw() }
```

Comment:
If you want something cool we can use the operator `&&` in place of `if ()`:
`function drawIf() { turtle.redraw && draw(); }`

Reply:
True, but I'm afraid bjpop, wouldn't like that. `&&` is a good idea if we want to minify the file, but the source is intended for humans, so `if` is better because it's "more explicit" (despite the fact that a lot of people know how `&&` works)

Reply:
> (despite the fact that a lot of people know how `&&` works)

Knowing how `&&` short-circuit works is very important for any JS & Python devs.
I wouldn't trust 1 who didn't know that.

Regardless, you've forgotten to end your modified `if ()` w/ `;`! Current 1 has it btW.

Reply:
> Knowing how `&&` short-circuit works is very important for any JS & Python devs.

I agree

> I wouldn't trust 1 who didn't know that.

lmao, I kinda agree

> Regardless, you've forgotten to end your modified `if ()` w/ `;`! Current 1 has it btW.

True. I thought that since the line ends with `}` the `;` is no longer needed. But it seems bjpop likes semicolons, so I should add it (I guess?)

Reply:
> , so I should add it (I guess?)

Well, Java would refuse to compile w/o that `;`. In JS `;` is kinda optional but to few edge cases.

Reply:
Ok I'm gonna add it lol